composer.json

```diff
@@ -18,7 +18,8 @@
         "php": "^7.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.2"
+        "phpunit/phpunit": "^6.2",
+        "symfony/process": "^2.x || ^3.3 || ^4"
```

Comment:
ah, almost ;P copy-paste of mine won't work ;)
`^3.3 || ^4` will work, but I don't know which 2.x you need in your work